app/docs/0.13.x/proxy.md

```diff
@@ -934,30 +980,33 @@ HTTP/1.1 200 OK
 ...
 ```
 
+When establishing the connection and negotiating the SSL handshake, if your
+client sends `ssl-example.com` as part of the SNI extension, Kong will serve
+the `cert.pem` certificate previously configured.
+
 [Back to TOC](#table-of-contents)
 
 ### The `https_only` property
 
-If you wish an API to only be served through HTTPS, you can do so by enabling
+If you wish a Route to only be served through HTTPS, you can do so by enabling
```

Comment:
I believe this whole section needs to be redone. There's no `https_only` property in routes. Instead we have the new `protocols` property, which accomplished similar goal, and more (it allows having `http only` as well).